config/routes.rb

```diff
@@ -2546,6 +2546,7 @@
         db_backup
         db_backup_form_field_changed
         db_gc_collection
+        db_list
```

Comment:
How is this related?

Reply:
That was an accidental add that got in somehow with switching between branches, I will remove that